README.md

```diff
@@ -29,61 +29,31 @@ Required env variables:
 
-#### GET request example
+#### Steps to run integration tests
 
-- Send a GET request with challenge phrase to the lambda function URL:
-
-  ```bash
-  pipenv run python -c "from lambdas.helpers import send_get_to_lambda_function_url; print(send_get_to_lambda_function_url('your challenge phrase'))"
-  ```
-
-  Observe output: `your challenge phrase`
-
-#### POST request examples
-
-- Send a POST request mimicking a webhook POST (not a POD or TIMDEX export job)
-
-  ```bash
-  pipenv run python -c "from lambdas.helpers import send_post_to_lambda_function_url, SAMPLE_WEBHOOK_POST_BODY; print(send_post_to_lambda_function_url(SAMPLE_WEBHOOK_POST_BODY))"
-  ```
-
-  Observe output: `Webhook POST request received and validated, no action taken.`
-
-- Send a POST request mimicking a POD export job webhook
-  _Note_: sending a request that mimics a POD export JOB_END will trigger the entire POD workflow, which is fine _in Dev1 only_ for testing.
-
-  Add the following to your .env:
-  - `VALID_POD_EXPORT_DATE=<the date of a POD export with files in the Dev1 S3 export bucket, in "YYYY-MM-DD" format>` Note: if it's been a while since the last POD export from Alma sandbox, there may be no files in the Dev1 S3 export bucket and you may need to run the publishing job from the sandbox.
-
-  ```bash
-  pipenv run python -c "from lambdas.helpers import send_post_to_lambda_function_url, SAMPLE_POD_EXPORT_JOB_END_WEBHOOK_POST_BODY; print(send_post_to_lambda_function_url(SAMPLE_POD_EXPORT_JOB_END_WEBHOOK_POST_BODY))"
-  ```
-
-  Observe output: `Webhook POST request received and validated, PPOD pipeline initiated.` and then check the Dev1 ppod state machine logs to confirm the entire process ran!
-
-- Send a POST request mimicking a TIMDEX export job webhook
-  _Note_: sending a request that mimics a TIMDEX export JOB_END will trigger the entire TIMDEX workflow, which is fine _in Dev1 only_ for testing.
-
-  Add the following to your .env:
-  - `VALID_TIMDEX_EXPORT_DATE=<the date of a TIMDEX export with files in the Dev1 S3 export bucket, in "YYYY-MM-DD" format>` Note: if it's been a while since the last TIMDEX export from Alma sandbox, there may be no files in the Dev1 S3 export bucket and you may need to run the publishing job from the sandbox.
-
-  ```bash
-  pipenv run python -c "from lambdas.helpers import send_post_to_lambda_function_url, SAMPLE_TIMDEX_EXPORT_JOB_END_WEBHOOK_POST_BODY; print(send_post_to_lambda_function_url(SAMPLE_TIMDEX_EXPORT_JOB_END_WEBHOOK_POST_BODY))"
-  ```
+Update docker image to ensure local changes are deployed to `Dev1`:
+```shell
+make publish-dev
+make update-lambda-dev
+```
 
-  Observe output: `Webhook POST request received and validated, TIMDEX pipeline initiated.` and then check the Dev1 timdex state machine logs to confirm the entire process ran!
+Run tests against deployed assets:
+```shell
+make test-integration
+```
 
 ## Running locally with Docker
```

Comment:
I would also enumerate the steps for the section "Running locally with Docker":

1. Build the container
   ```
   ```
2. Run the default handler for the container. 
   ```
   ```
   Depending on what you're testing, you may need to pass `-e WORKSPACE=dev` and/or other environment variables as options to the `docker run` command.
3. Post to the container. 
   ```
   ```
4. Observe output. Note: running the above with no env variables passed should result in an exception.


(Minor but something that would improve the README!)